templates/workspaces/base/terraform/variables.tf

```diff
@@ -32,7 +32,7 @@ variable "deploy_app_service_plan" {
 
 variable "app_service_plan_sku" {
   type        = string
-  default     = "P1v3"
+  default     = "P1v2"
```

Comment:
Can we leave this as is? We seem to keep flipping?

Reply:
CHanged it back to P1v3